index.js

```diff
@@ -60,15 +60,40 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
+const header = document.querySelector("h1#header")
 function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+  if (element.target.style.color === "red") {
+    element.target.style.color = "black"
   } else {
-    element.style.color = "red"
+    element.target.style.color = "red"
   }
 }
 
+header.addEventListener("click", toggleColor)
```

Comment:
Nice work updating this callback to work with the event listener! I would suggest re-naming the parameter to `event` so it's clearer that this function is working with the event object:

```js
const header = document.querySelector("h1#header")

function toggleColor(event) {
  if (event.target.style.color === "red") {
    event.target.style.color = "black"
  } else {
    event.target.style.color = "red"
  }
}

header.addEventListener("click", toggleColor)
```